<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptStatus</class>
 <widget class="QWidget" name="OptStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptStatusUI</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Auto status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QGridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sb_asOffline">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="lb_asOffline">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Minutes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lb_asXa">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Minutes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sb_asXa">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lb_asAway">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Minutes</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="ck_asOffline">
          <property name="text">
           <string>Auto offline after:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="ck_asXa">
          <property name="text">
           <string>Auto XA after:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="ck_asAway">
          <property name="text">
           <string>Auto away after:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sb_asAway">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ck_asAway</tabstop>
  <tabstop>sb_asAway</tabstop>
  <tabstop>ck_asXa</tabstop>
  <tabstop>sb_asXa</tabstop>
  <tabstop>ck_asOffline</tabstop>
  <tabstop>sb_asOffline</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ck_asOffline</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asOffline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asAway</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asAway</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asXa</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asXa</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
